app/controllers/travisci-vs-jenkins.js

```diff
@@ -0,0 +1,15 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import config from 'travis/config/environment';
+
+export default Controller.extend({
+  router: service(),
+
+  caseStudyUrl: config.urls.caseStudy,
+
+  goToContactForm() {
+    this.router.transitionTo('plans', {
+      queryParams: { section: 'contact' },
+    });
+  }
```

Comment:
It seems like this action is only used in templates, is it? If so, then probably it could be converted to `LinkTo`, what do you think @sorta? 
```hbs
<LinkTo @route='plans' @query={{hash section='contact'}}>
```

Reply:
Yeah that makes sense, I'll make that change.

Reply:
Change made :)